backend/assignments/2/src/main/java/com/caching/exception/LocationNotFoundException.java

```
@@ -0,0 +1,4 @@
+package com.caching.exception;
+
+public class LocationNotFoundException extends RuntimeException{
```

Exceptions are not handled properly